### DISCUSSION TITLE
License file has a typo.

### DISCUSSION THREAD

**Discussion by steve-h** [fatbobman/SwiftUIOverlayContainer#17] [General]
Copyright (c) 2022 东坡肘子 ( Fatobman ) is in the version that I was looking at. I do not really think a PR is appropriate other than by the owner. A change would in theory need to be propagated to all the users that include a copy. I have not seen tooling to keep these license texts up-to-date. Do as you see fit. I do not know the answer, but assume that I will propagate the typo. In a newspaper the notation (sic) would be added to denote the typo in the original.

**Comment by fatbobman** [comment]
Thank you very much, I will correct the typo.
